sql/2024/fonts/performance/fonts_format_file.sql

```diff
@@ -0,0 +1,28 @@
+-- Section: Performance
+-- Question: Which file formats are used?
+
+-- INCLUDE ../common.sql
+
+SELECT
```

Comment:
The numbers are *very* different from 2022, has anything else changed?

Reply:
Changed the normalization to “Fonts on pages.”